tests/common_tools.py

```diff
@@ -190,10 +191,16 @@ def create_valid_ticket_for_user_and_fare(user, fare=None):
     return ticket
 
 
-def get_default_conference(end=None):
+def get_default_conference(end=None, voting_start=None, voting_end=None):
     if not end:
```

Comment:
Good job, but what do think about a ConferenceFactory with [django_get_or_create](https://factoryboy.readthedocs.io/en/latest/orms.html#factory.django.DjangoOptions.django_get_or_create)

Example:

```python
class ConferenceFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = 'app.conference'
        django_get_or_create = ('name',)

    name = 'EuroPython 2020'
    code = settings.CONFERENCE_CONFERENCE

    conference_start = factory.Faker('future_date', end_date="+30d")
    conference_end = factory.LazyAttribute(lambda obj: obj.conference_start + timedelta(days=7))
    cfp_start = factory.LazyAttribute(lambda obj: obj.conference_start - timedelta(days=33))
    cfp_stop = factory.LazyAttribute(lambda obj: obj.cfp_start + timedelta(days=3))
    voting_start = factory.LazyAttribute(lambda obj: obj.cfp_start + timedelta(days=3))
    voting_end = factory.LazyAttribute(lambda obj: obj.voting_start + timedelta(days=3))
``` 

you could use it like that:

```python
def get_default_conference(end=None, voting_start=None, voting_end=None):
    parameters = {
        'name': 'EuroPython 2020',
        'voting_start': voting_start,
        'voting_end': voting_end,
        'end': end,
    }
    return ConferenceFactory(**{k: v for k, v in parameters.items() if v is not None})
```

Reply:
That's a good idea, thanks! going to update that.